src/app/core/component/graph-display/graph-display.component.ts

```diff
@@ -210,18 +255,35 @@ export class GraphDisplayComponent implements AfterContentInit, OnChanges {
   }
 
   nodeClicked(event) {
```

Comment:
A function does something, i.e., a name should indicate this. Although this is an event listener it is much more readable of the name indicates what's going on: e.g., handleNodeClickedEvent might be better.

Reply:
Done